src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiExtSqlBase.g4

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2021 Rikai authors
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// Grammar for Rikai-extensions.
+grammar RikaiExtSqlBase;
+
+singleStatement
+    : statement ';'* EOF
+    ;
+
+statement
+    : CREATE (OR REPLACE)? MODEL model=qualifiedName
+	  (OPTIONS optionList)?
```

Comment:
Not related to this PR: do we want to have limited list of options?

Reply:
You meant the limited choices? I dont have strong options at the moment.  I do not feel that I can come up a good set of options now tho.